src/main/java/eu/dissco/orchestration/backend/service/ProvenanceService.java

```diff
@@ -75,6 +85,16 @@ private CreateUpdateTombstoneEvent generateCreateUpdateTombStoneEvent(
         ));
   }
 
+  private static String getRdfsComment(ProvActivity.Type activityType) {
```

Comment:
I would use a switch in this case (as we cover all enums, default is not necessary)